package.json

```diff
@@ -80,7 +80,7 @@
     "m-react-splitters": "1.2.0",
     "moment": "2.24.0",
     "react": "16.13.1",
-    "react-ace": "6.4.0",
+    "react-ace": "9.1.3",
```

Comment:
With this upgrade, have the foldings still work? I am thinking of the scenario where the status field is present in the yaml.

Reply:
I haven't tested it, going to see if I can take some screenshots.